en/vespa-feed-client.html

```diff
@@ -28,11 +28,17 @@ <h3>Java library</h3>
 <h3>Command line client</h3>
 <p>Two alternatives:</p>
 <ul>
-  <li>Download <a href="https://search.maven.org/artifact/com.yahoo.vespa/vespa-feed-client-cli">
-    vespa-feed-client <strong>zip</strong> artifact</a> from Maven Central.
   <li>Install <a href="build-install-vespa.html"><em>vespa-clients</em>/<em>vespa</em></a> RPM package.</li>
+  <li>Download <a href="https://search.maven.org/artifact/com.yahoo.vespa/vespa-feed-client-cli">
+    vespa-feed-client <strong>zip</strong> artifact</a> from Maven Central.</li>
 </ul>
-
+<p>Download example:</p>
+<pre>
+$ F_REPO="https://repo1.maven.org/maven2/com/yahoo/vespa/vespa-feed-client-cli" && \
+  F_VER=$(curl -Ss "${F_REPO}/maven-metadata.xml" | sed -n 's/.*&lt;release&gt;\(.*\)<.*>/\1/p') && \
```

Comment:
Forgot to XML-quote the second `<>` pair, though ...

Reply:
argh. #2261  to fix that. thanks!